src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -222,7 +235,8 @@
                 WITHDRAW_CLAIM,
                 DISCONTINUE_CLAIM,
                 CASE_PROCEEDS_IN_CASEMAN,
-                AMEND_PARTY_DETAILS
+                AMEND_PARTY_DETAILS,
+                ADD_CASE_NOTE
             )
         ),
         entry(
```

Comment:
In terms of access control I believe we are missing the add note event for the following two flow states: CLAIM_DISMISSED_PAST_CLAIM_NOTIFICATION_DEADLINE, CLAIM_DISMISSED_PAST_CLAIM_DETAILS_NOTIFICATION_DEADLINE

I have updated the diagram and user story ACs to reflect this:
![image](https://user-images.githubusercontent.com/138029/125116165-361d7e80-e0e4-11eb-9256-28a06df2e6b9.png)